apps/studio/components/interfaces/Auth/SiteUrl/SiteUrl.tsx

```diff
@@ -41,26 +41,27 @@ const SiteUrl = () => {
     'custom_config_gotrue'
   )
 
-  const siteUrlForm = useForm({
-    resolver: yupResolver(schema),
+  const form = useForm<z.infer<typeof schema>>({
+    resolver: zodResolver(schema),
     defaultValues: {
       SITE_URL: '',
     },
   })
 
   useEffect(() => {
     if (authConfig && !isUpdatingSiteUrl) {
-      siteUrlForm.reset({
+      form.reset({
         SITE_URL: authConfig.SITE_URL || '',
       })
     }
   }, [authConfig, isUpdatingSiteUrl])
```

Comment:
might be able to remove this `useEffect` by leveraging the `key` property instead